include/dolphin/gd/GDBase.h

```diff
@@ -40,6 +40,10 @@ inline u8* GDGetCurrPointer() {
     return __GDCurrentDL->ptr;
 }
 
+inline s32 GDGetCurrOffset() {
```

Comment:
Is this real? I thought I only saw GDGetGDLObjOffset in the debug map

Reply:
I'm working off of the twilight princess debug rom. Some areas are very similar, J3DMatBlock included. GDGetCurrOffset does exist in the wind waker debug map, so I see no reason not to use it.